spec/text_confirmation_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'text_confirmation'
+
+describe TextConfirmation do
+
+  subject(:text_confirmation) { described_class.new }
+  let (:order) { double :order }
+
+  it 'should create and send a text message' do
+    expect(text_confirmation).to receive(:order_finished).with(order)
```

Comment:
It feels like this test isn't testing very much.  The test calls `order_finished` and then checks it was called.  Have a think about what you actually want to check.  Happy to talk this through - grab me if you want!